packages/yoroi-extension/app/containers/wallet/Wallet.js

```diff
@@ -218,8 +243,9 @@ class Wallet extends Component<AllProps> {
         showAsCard
       >
         {warning}
-        {this.renderOverlay()}
         {this.props.children}
+        {this.renderOverlay()}
```

Comment:
Why are we changing the order of `{this.renderOverlay()}` to after `{this.props.children}`?

Reply:
No specific reason. I think this diff was made while I was testing something.